### DISCUSSION TITLE
Can we support safari 14.4.2?

### DISCUSSION THREAD

**Discussion by SunnyCheung-cp** [Milkdown/milkdown#1304] [Q&A]
I'm using safari14.4.2 and it looks blank
![IMG_0308](https://github.com/Milkdown/milkdown/assets/105030733/e5c5cec2-8134-45e3-b6d7-f9d60c8587b9)

**Comment by omarmir** [comment]
I had the same issue. If you check your web inspector do you see invalid regular expression? Maybe related to this issue: https://github.com/Milkdown/milkdown/issues/1703
